### PR TITLE
Exercises ppj

### PR DESCRIPTION
**Pull request by lukaszkutylowski**

### CODE REVIEW

**Review by MatDls — CHANGES_REQUESTED** [review]
too many files for one PR. please, split it.

**Comment by MatDls** [comment]
you work pretty faat, that's super cool:)

**Comment by lukaszkutylowski** [comment]
Ok, I read about split PRs but I think the most advantageous is reupload files in new commit and create pull request. In this situation create new branches and delete old branch to split one RP is pointless. I'm going to work

**Comment by lukaszkutylowski** [comment]
I should add one file (or main file + method file) in another branch? I cannot add for example 30 files in one branch? (branch = pull request, i see). Because only way to split pull request is create new branches

File: ExercisesPPJ/ExercisePPJ_04_4.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	int x = in.nextInt();
+	
+	System.out.println("number is: "+x);
+	
+	int i = 0;
+	if(x < 10) {
+		
+		System.out.println("input number is less than 10");
+	}
+	else if((x < 100) && (x >= 10)) {
```
Comment (MatDls) [inline ExercisesPPJ/ExercisePPJ_04_4.java]: those parenthesis are not necessary, but they are ok

File: ExercisesPPJ/ExercisePPJ_04_2.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	
+	System.out.print("wynik operacji 'System.out.println(aChar + bInt);' to ");
+	System.out.println(aChar + bInt);
+	System.out.print("wynik operacji 'System.out.println(bInt + aChar);' to ");
+	System.out.println(bInt + aChar);
+	System.out.print("wynik operacji 'System.out.println(cFloat + dDouble);' to ");
+	System.out.println(cFloat + dDouble);
+	System.out.print("wynik operacji 'System.out.println(eByte + bInt);' to ");
+	System.out.println(eByte + bInt);
+}
```
Comment (MatDls) [inline ExercisesPPJ/ExercisePPJ_04_2.java]: indentation, please:)

Comment (MatDls) [reply ExercisesPPJ/ExercisePPJ_04_2.java]: indentation - in may places:)
Please, be careful about that:)

**Review by MatDls — CHANGES_REQUESTED** [review]
Gnerally, it looks nice. 
But, for practice please:
Split this pull int two.
One with Exc. 04_* other wiht Exc. 05_*
I think you will fugure out how to do that:)

**Pull request closed without merging**